libs/base/ui/structure/collapsible/collapsible.component.ts

```diff
@@ -67,8 +80,19 @@ export class CollapsibleComponent
     }
   }
 
-  protected onClick(): void {
-    this.isManuallyOpened = true;
+  protected syncState(store = false): void {
+    if (!this.persistedStateKey) return;
+
+    if (store) {
+      if (this.isManuallyOpened) {
+        this.uiController.set(this.persistedStateKey, this.open);
```

Comment:
shouldn't we prefix key with component name? as the controller could be used in different components, so there is a risk of key collision? (or developer has to care about it)